### PR TITLE
Prepare for Renovate-Bot

### PR DESCRIPTION
**Pull request by weierophinney**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | yes
| QA            | yes

### Description

This patch prepares for the Renovate-Bot, performing the following changes:

- **DROPS** support for PHP 7.3
- Updates to Laminas CI 2.3 series (disabling strict types sniff), and  applies automated and manual phpcs fixes as flagged by the Laminas CS
- Adds a platform PHP configuration settings
- Updates the CI and release workflows to use the automated workflows

### CODE REVIEW

File: src/BlockCipher.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ class BlockCipher
      *
      * @var SymmetricPluginManager
      */
-    protected static $symmetricPlugins = null;
+    protected static $symmetricPlugins;
```
Comment (Ocramius) [inline src/BlockCipher.php:48]: This component is very security sensitive: perhaps worth asking @ezimuel  for a review, before changing CS?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: src/BlockCipher.php (hunk 1 above)
Comment (weierophinney) [reply src/BlockCipher.php:48]: I've audited the changes, and other than formatting, the changes include:

- grouping removals (e.g. removing parens from `( ... )` expressions) where they had no effect
- assignment to null removed in property declarations (default is `null` anyways, and these are tested either using `isset()` or truthiness tests)
- ternary operations converted to null coalesce where possible
- `list()` operations become `[]`
- strict comparisons added when the value types are known (always integer, string, or constant comparisons)
- nullable operator provided when default value of a typed variable is `null`
- `__CLASS__` references become `self::class`

I can certainly wait for @ezimuel to weigh in if he has time, but I was pretty cautious (it's part of why I excluded the addition of the `declare(strict_types=1)` requirement in the ruleset).

**Comment by ezimuel** [comment]
@Ocramius , @weierophinney it looks good to me. I didn't find security potential changes.

**Review by Ocramius — APPROVED** [review]
Excellent, thanks @weierophinney and @ezimuel!

**Pull request merged**